pdns/iputils.hh

```diff
@@ -276,7 +276,7 @@ union ComboAddress {
       return "["+toString() + "]:" + std::to_string(ntohs(sin4.sin_port));
   }
 
-  void truncate(unsigned int bits);
+  void truncate(unsigned int bits) noexcept;
```

Comment:
my reading of http://en.cppreference.com/w/cpp/language/noexcept_spec does not tell me that `noexcept` -enforces- anything. There is no downside to having the keyword there, of course.